contracts/dex/src/exchanges/kujira.rs

```diff
@@ -0,0 +1,317 @@
+use crate::{
+    error::DexError,
+    traits::{
+        command::{DexCommand, Fee, FeeOnInput, Return, Spread},
+        identity::Identify,
+    },
+    util::coins_in_assets,
+};
+use abstract_core::objects::PoolAddress;
+use abstract_sdk::cw_helpers::wasm_smart_query;
+use cosmwasm_std::{
+    wasm_execute, Addr, Coin, CosmosMsg, Decimal, Decimal256, Deps, StdError, StdResult, Uint128,
+};
+use cw_asset::{Asset, AssetInfo, AssetInfoBase};
+use kujira::{
+    bow::{
+        self,
+        market_maker::{ConfigResponse, PoolResponse},
+    },
+    fin,
+};
+pub const KUJIRA: &str = "kujira";
+
+// Source https://docs.rs/kujira/0.8.2/kujira/
+pub struct Kujira {}
+
+impl Identify for Kujira {
+    fn name(&self) -> &'static str {
+        KUJIRA
+    }
+    fn over_ibc(&self) -> bool {
+        false
+    }
+}
+
+impl DexCommand for Kujira {
```

Comment:
Here you also need to only feature flag the DexCommand implementation part and not the Identify part (same reson, for IBC intergation"

Reply:
done